src/main.rs

```diff
@@ -14,15 +15,14 @@ use rand::distributions::Uniform;
 use rand::seq::IteratorRandom;
 use rand::{thread_rng, Rng};
 
+use assigner::hash;
+use assigner::message::{Assignment, Get};
 use task::face::Expression;
 use task::message::{Request, Response};
 
 const BUFFER_SIZE: usize = 256;
 
-fn get_task_addr() -> String {
-    // String::from("54.183.196.119:3333")
-    String::from("localhost:3333")
-}
+const ASSIGNER_ENDPOINT: &str = "184.169.220.191:4333";
```

Comment:
nit: let's keep naming consistent throughout our repos and say ASSIGNER_ADDRESS or ASSIGNER_ADDR